### ISSUE TITLE
WebView control shows a blank page.

### ISSUE THREAD

**Issue by golrokhka** [MicrosoftEdge/WebView2Feedback#2115]
Hi there,

I have a Webview control in my application which works perfectly fine while running the app in Vs.net. Once I deploy the app and I haven't installed webviewer yet, it shows a link to the page instead of loading the page in the browser which makes sense! Once I install the webviewer runtime, it shows just a black page! What could cause this issue?
I have tried evergreen bootstrapper and standalone installer and the result are the same for both.

Do you have any suggestions on how I can fix this?

Thank you,
Goli

**Comment by champnic** [comment]
Hey @golrokhka - Can you add some info like which UI framework and version you are using for your app, WebView2 SDK version, and WebView2 runtime version?

How are you deploying your app? You may be missing the WebView2Loader.dll:
https://docs.microsoft.com/en-us/microsoft-edge/webview2/concepts/distribution#files-to-ship-with-the-app

**Comment by golrokhka** [comment]
> Hey @golrokhka - Can you add some info like which UI framework and version you are using for your app, WebView2 SDK version, and WebView2 runtime version?
> 
> How are you deploying your app? You may be missing the WebView2Loader.dll: https://docs.microsoft.com/en-us/microsoft-edge/webview2/concepts/distribution#files-to-ship-with-the-app

Hi @champnic,

Our app is a desktop app_ WPF/WinForms. Also, we're using "Inno setup".  WebView2Loader.dll is there. I've checked it.
One thing that I noticed is we're using an old version of Webviewer in our app but we have the latest in our installer. Still not sure if this is what causes the issue but worth trying to upgrade the version to the latest one and will see...
The other thing I noticed is when we uninstall the Webview, it's not gonna clean out both reg keys from the local machine and current user. The app checks the reg key to figure out if the wbview2 has been installed or not. So it assumes yes, it's there. Yet, it's not! So the page would be blank in this case too but this rarely happens.
